iscsi/initiator.go

```
@@ -166,6 +179,33 @@ func IsTargetLoggedIn(ip, target string, ne *util.NamespaceExecutor) bool {
 	return found
 }
 
+func manualScanSession(ip, target string, ne *util.NamespaceExecutor) {
+	opts := []string{
+		"-m", "node",
+		"-T", target,
+		"-p", ip,
+		"--rescan",
+	}
+	_, err := ne.ExecuteWithTimeout(sessionScanTimeout, iscsiBinary, opts)
+	if err != nil {
+		logrus.Warnf("Failed to manually rescan iscsi session of target %v:%v", target, ip)
+	}
+}
+
+func getIscsiNodeSessionScanMode(ip, target string, ne *util.NamespaceExecutor) string {
+	opts := []string{
+		"-m", "node",
+		"-T", target,
+		"-p", ip,
+		"-o", "show",
+	}
+	output, err := ne.ExecuteWithTimeout(getScanModeTimeout, iscsiBinary, opts)
```

Do not ignore the error. Log it if we're not planning to deal with it.